swan-lake/integration-tutorials/content-based-message-routing.md

```diff
@@ -164,7 +164,11 @@ Follow the instructions given in this section to develop the service.
         boolean confirmed;
         string appointmentDate;
     };
+    ```
+
+    Similarly, generate the `ReservationRequest` record type and update the `hospital_id` field with the `HospitalId` type. Remove a one `Patient` record if it gets generated twice.
```

Comment:
```suggestion
    Similarly, generate a record corresponding to the request payload (e.g., `ReservationRequest`) and update the `hospital_id` field to be of the `HospitalId` type that was defined previously. Remove the duplicate `Patient` record since the same type has already been generated.
```

Reply:
+1. Also, shall we put "Remove the duplicate `Patient` record if the same type has already been generated" (with `if`) since this bug might be fixed in future.

Reply:
+1.